### PR TITLE
Completed css-canvas-medical challenge 1.

### PR DESCRIPTION
**Pull request by AnthonyVNgo**
screencapture-127-0-0-1-5500-css-canvas-medical-index-html-2022-03-20-00_01_58

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Anthony! The capitalization is wrong on your "Clinicians Prefer Canvas" heading, please revise and submit a new screenshot with your revision.

**Comment by AnthonyVNgo** [comment]
screencapture-127-0-0-1-5500-css-canvas-medical-index-html-2022-03-22-11_58_03

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            <img src="images/logo-white.png" alt="logo" class="logo border-box fit-content tb-padding-small">
+            <p>
+              2307 Irving Street, Suite 228 <br>
+              San Francisco CA 94122 <br>
+              (800) 370-1416 <br>
+              info@canvasmedical.com
+            </p>
+          </div>
+
+          <div class="column-fourths"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html]: You should never have empty `column` elements in your code. Instead, you should make the `column-fourth` that precedes this element into a `column-half`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**